### ISSUE TITLE
“Prompt_tone” entity keeps turning back on after being unavailable

### ISSUE THREAD

**Issue by Post404** [georgezhao2010/midea_ac_lan#407]
### HA core version

2023.12.3

### Intergration version

0.3.22

### Device type and model

Air Conditioner 00000Q13 (0) by Midea

### Used App

Midea Air

### The description of problem

After being unavailable the prompt tone entity keeps turning on. Sometimes it also turns on randomly.

### The logs

No response

**Comment by zenguru84** [comment]
same here. 
the same issue on 2 devices.
frustrating when automations run at night and the AC beeps.

**Comment by akiriik** [comment]
I just made a simple workaround with automation... If Tone turns on -> Turn it off.

**Comment by sanderlv** [comment]
May I ask. I recently switched to this integration prior the other on available. Since I have this "beep" randomly but frequently coming out of my comfee even when it is totaly turned off (no leds are on)!? 

Is this the issue you are also experiencing and if so can you please specifiy more what you have done to solve this?![20240108_201111.jpg](https://github.com/georgezhao2010/midea_ac_lan/assets/16528916/22a1539b-c0c0-4ad8-97eb-e8175f5b733f)

**Comment by Post404** [comment]
I don't think it beeps when it is completely off. It does beep when it is turned off, maybe that is what is happening.
I only partially solved it by turning off the beep everytime before there is a command send to the airco. It is beeping less now, but it doesn't always work. It used to work perfect on the earlier integration, but that one was no longer maintained.
